### DISCUSSION TITLE
Refresh token rotation for Next.js app router

### DISCUSSION THREAD

**Discussion by rikurainio** [nextauthjs/next-auth#10470] [Help]
Hey. Is this under work? 
Is there some workaround to implement token refreshing on Next.js + Auth.js app?
The JWT session strategy works fine other than that.
Thanks for this awesome library.

**Comment by Enkratia** [comment]
Offical documentation is not good for your purposes? https://authjs.dev/guides/basics/refresh-token-rotation

**Reply by rikurainio** [reply]
![chrome_eBWSLbwPd7](https://github.com/nextauthjs/next-auth/assets/76222709/27168495-d203-4f33-9e65-97a761cfacff)

**Comment by stdunbar** [comment]
> Offical documentation is not good for your purposes? https://authjs.dev/guides/basics/refresh-token-rotation

Yeah, it 404's
